functions/exercise13/script.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+function getFactorsOfPositiveInteger(number) {
+    var array = [];
+    for (var i = 1; i <= number; i++)
+        if (number % i === 0)
```

Comment:
Please always add brackets for action executed in if statement. 
Please turn on this verfication in ESLint:
http://eslint.org/docs/rules/curly